src/Servers/IIS/IIS/test/Common.FunctionalTests/Utilities/IISTestSiteCollection.cs

```diff
@@ -13,17 +13,4 @@ public class IISTestSiteCollection : ICollectionFixture<IISTestSiteFixture>
     {
         public const string Name = nameof(IISTestSiteCollection);
     }
-
-    [CollectionDefinition(Name)]
-    public class OutOfProcessTestSiteCollection : ICollectionFixture<OutOfProcessTestSiteFixture>
-    {
-        public const string Name = nameof(OutOfProcessTestSiteCollection);
-    }
-
-    [CollectionDefinition(Name)]
-    public class OutOfProcessV1TestSiteCollection : ICollectionFixture<OutOfProcessV1TestSiteFixture>
```

Comment:
Are we losing test coverage by removing this?

Reply:
Nope. OutOfProcV1 was removed from 3.0.